src/journal/journal.c

```diff
@@ -393,9 +408,44 @@ static int is_class_prefix(char *class)
         return (strcmp((char *)(class + class_len - 2), "/*") == 0) ? 1 : 0;
 }
 
+/**
+ * Print records content
+ *
+ * @param record_id Unique record identifier
+ *
+ */
+static void print_record(char *record_id)
+{
+        int rc = 0;
+        size_t read = 0;
+        char buff[MID_BUFF] = { '\0' };
+        char *filepath = NULL;
+        FILE *recordfp = NULL;
+
+        rc = asprintf(&filepath, "%s/%s", RECORD_RETENTION_DIR, record_id);
+        if (rc == -1) {
+                // just bail out, there are worse problems
+                return;
+        }
+
+        recordfp = fopen(filepath, "r");
+        if (!recordfp) {
+                perror("Error");
```

Comment:
A more specific string should be supplied here, or else you will not know where in the code in the error occurred. Also, any reason for not using `telem_perror()`?